collection_requirements.rst

```diff
@@ -175,15 +175,13 @@ CI Testing
 ===========
 
 * You MUST run ``ansible-test sanity`` from the `latest stable ansible-base/ansible-core branch <https://github.com/ansible/ansible/branches/all?query=stable->`_. 
-* You MUST run CI against all versions of ``ansible-base``/``ansible-core`` that the collection supports.
 * You SHOULD suggest to *additionally* run ``ansible-test sanity`` from the ansible/ansible ``devel`` branch so that you find out about new linting requirements earlier.
 * The sanity tests MUST pass.
 
   * Adding some entries to the ``test/sanity/ignore*.txt`` file is an allowed method of getting them to pass. 
   * All entries in ignores.txt MUST have a justification in a comment in the ignore.txt file for each entry.  For example ``plugins/modules/docker_container.py use-argspec-type-path # uses colon-separated paths, can't use type=path``.
   * Reviewers can block acceptance of a new collection if they don't agree with the ignores.txt entries.
 
-* All CI tests MUST run against every PR & commit to the repo.
```

Comment:
MUST run, SHOULD pass. We understand that occasionally things outside of your control may break your tests.

Reply:
Is there a way to achieve this with GitHub Actions or AZP? If you push several commits at once, right now (at least for GHA and Shippable) tests are only run for the last commit, not for the ones before.

Reply:
tadeboro also brought up that this could be an undue burden to maintainers.  For instance, a company that pays for every CI run and does not squash their commits could end up paying a lot to do this.  We don't really care about the intermediate commits when we think about ci being run.  Is there another way we could phrase this that would capture what we really want?  Like "CI must be run on the head of every PR and the repository branches that you release from"?

We should probably add the note about SHOULD pass with the explanation you give as well.  That seems like an important exception to mention.

Reply:
Ah, I hadn't considered that.
OK to change to SHOULD

Reply:
```
* All CI tests MUST run against every pull request (last pushed commit to the repository) and MUST pass before merge with the exception when something outside of maintainers control breaks them.
* All CI tests MUST pass for the commit that releases the collection.
```
? or something like that

Reply:
@abadger Hows does the above suggestion sound to you?

Reply:
@abadger ^ ? If it's ok, i could add

Reply:
@Andersson007 it's already here (in slightly modified form): https://github.com/ansible-collections/overview/pull/137/files#diff-dacab21f27a6aa218b8dcaafcf6f5685df0ab52f231afa16e7a9ab5284b72c68R200-R201

Reply:
@felixfontein you were 4 minutes faster than me:) I've just found that in the meeting summary